staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -139,6 +139,33 @@ func NewAvailableConditionController(
 	return c
 }
 
+func setConditionAndUpdateStatus(client apiregistrationclient.APIServicesGetter, apiService *apiregistration.APIService, newCondition apiregistration.APIServiceCondition) error {
+	orig := apiService.DeepCopy()
+	apiregistration.SetAPIServiceCondition(apiService, newCondition)
+	if !reflect.DeepEqual(orig, apiService) {
```

Comment:
invert the check and return early if equal to unnest?

Reply:
done